### PR TITLE
Upgrade SDK and migrate network settings

### LINKED ISSUE #420: Upgrade SDK to work with node 1.5.0 and explorer backend 1.9.0
Following this: https://github.com/alephium/js-sdk/issues/130

### LINKED ISSUE #429: Change testnet endpoints and write migration script
Change endpoints to:

```
https://wallet.testnet.alephium.org/
https://backend.testnet.alephium.org/
```

### PR DESCRIPTION
**Pull request by nop33**
Closes #429
Closes #420

Waiting for:
- [x] https://github.com/alephium/js-sdk/pull/131 to be merged, and sdk `v0.3.0` to be released
- [x] clarification on which should be the new mainnet endpoints

### CODE REVIEW

File: src/utils/settings.ts [hunk 1]
```diff
@@ -115,20 +115,37 @@ export const migrateDeprecatedSettings = (): Settings => {
...
+  if (
+    settings.network.nodeHost === 'https://mainnet-wallet.alephium.org' ||
+    settings.network.nodeHost === 'https://wallet-v18.mainnet.alephium.org'
+  ) {
+    migratedSettings.network.nodeHost = 'https://wallet-v15.mainnet.alephium.org'
+  } else if (
+    settings.network.nodeHost === 'https://testnet-wallet.alephium.org' ||
+    settings.network.nodeHost === 'https://wallet-v18.testnet.alephium.org'
+  ) {
+    migratedSettings.network.nodeHost = 'https://wallet-v15.testnet.alephium.org'
```
Comment (mvaivre) [inline src/utils/settings.ts:148]: Should we maybe add some tests for the network settings migration?

Comment (nop33) [reply src/utils/settings.ts:148]: Sounds like a great idea!

Comment (mvaivre) [reply src/utils/settings.ts:148]: So fast ⚡ !

**Review by mvaivre — APPROVED** [review]
👏

**Pull request merged**